test/AssetPool.test.js

```diff
@@ -285,26 +328,57 @@ describe("AssetPool", () => {
 
     context("when called via approveAndCall", () => {
       const amount = to1e18(100)
+      const abiCoder = ethers.utils.defaultAbiCoder
 
-      beforeEach(async () => {
-        await collateralToken
-          .connect(underwriter1)
-          .approveAndCall(assetPool.address, amount, [])
+      context("when passed correct parameters", () => {
```

Comment:
We could be more specific about what *correct parameters* are, for example: *when passed minimum amount to mint uint256 in extradata*.

Reply:
Done as part of 485362d